### ISSUE TITLE
I dont think you can use this sensor for flash measurements.

### ISSUE THREAD

**Issue by cranefist** [vpominchuk/lightmeter#6]
How can this work as a flash meter?

You code says its measuring time in milliseconds, and the fastest measurement this sensor can do is 16ms!

While camera flashes operate in 0.05-10ms range. There is no way you can use this sensor to measure camera flashes accurately. You would need to be able to take measurements in microseconds, and measure time in microsecond resolution.

**Comment by vpominchuk** [comment]
You are right it's not so accurate to measure flash light, especially in HS mode, but it works.
The main goal of this project is measuring an ambient light for film cameras and in this case this cell do his job well.
Measuring the flash light is a bonus and not always accurate.

**Comment by cranefist** [comment]
Ok, i was looking into measuring flash.

There is a video about flash durations here : https://www.youtube.com/watch?v=Ks0T1Utm0D8

And its simply too fast for microcontrollers, and for these types of lux meters. As flash light is about intensity & duration.
